mediapipe/calculators/ovms/openvinoinferencecalculator_test.cc

```diff
@@ -318,3 +322,410 @@ TEST_F(OpenVINOInferenceCalculatorTest, DISABLED_HandleEmptyPacketsWithSyncSet)
+                    }
+                    tag_to_output_tensor_names {
+                        key: "SUM"
+                        value: "sum"
+                    }
+                    }
+                }
+            )pb");
+    auto cc = absl::make_unique<CalculatorContract>();
+    cc->Initialize(calculator);
+    auto abslStatus = mediapipe::OpenVINOInferenceCalculator::GetContract(cc.get());
+    ASSERT_EQ(abslStatus.code(), absl::StatusCode::kOk) << abslStatus.message();
+}
+
+TEST_F(OpenVINOInferenceCalculatorTest, UnSupportedTypeTagToInputNamesOutputMismatch) {
+    auto calculator =
+        mediapipe::ParseTextProtoOrDie<CalculatorGraphConfig::Node>(
+            R"pb(
+                calculator: "OpenVINOInferenceCalculator"
+                input_side_packet: "SESSION:session"
+                input_stream: "INPUT1:in1"
+                input_stream: "INPUT2:in2"
+                output_stream: "SUM:out"
+                node_options: {
+                    [type.googleapis.com / mediapipe.OpenVINOInferenceCalculatorOptions]: {
+                    tag_to_input_tensor_names {
+                        key: "INPUT1"
+                        value: "input1"
+                    }
+                    tag_to_input_tensor_names {
+                        key: "INPUT2"
+                        value: "input2"
+                    }
+                    tag_to_output_tensor_names {
+                        key: "SUM1"
+                        value: "sum"
+                    }
+                    }
+                }
+            )pb");
+    auto cc = absl::make_unique<CalculatorContract>();
+    cc->Initialize(calculator);
+    auto abslStatus = mediapipe::OpenVINOInferenceCalculator::GetContract(cc.get());
+    ASSERT_EQ(abslStatus.code(), absl::StatusCode::kInternal) << abslStatus.message();
+}
+
+TEST_F(OpenVINOInferenceCalculatorTest, UnSupportedTypeTagToInputNamesInputMismatch) {
+    auto calculator =
+        mediapipe::ParseTextProtoOrDie<CalculatorGraphConfig::Node>(
+            R"pb(
+                calculator: "OpenVINOInferenceCalculator"
+                input_side_packet: "SESSION:session"
+                input_stream: "INPUT1:in1"
+                input_stream: "INPUT2:in2"
+                output_stream: "SUM:out"
+                node_options: {
+                    [type.googleapis.com / mediapipe.OpenVINOInferenceCalculatorOptions]: {
+                    tag_to_input_tensor_names {
+                        key: "INPUT3"
+                        value: "input1"
+                    }
+                    tag_to_input_tensor_names {
+                        key: "INPUT2"
+                        value: "input2"
+                    }
+                    tag_to_output_tensor_names {
+                        key: "SUM"
+                        value: "sum"
+                    }
+                    }
+                }
+            )pb");
+    auto cc = absl::make_unique<CalculatorContract>();
```

Comment:
Minor but we could wrap body of all those tests and just call
```suggestion

void verifyGetContract(const std::string& pbtxtContent, absl::StatusCode expectedStatusCode) {
    auto calculator = mediapipe::ParseTextProtoOrDie<CalculatorGraphConfig::Node>(pbtxtContent);
    auto cc = absl::make_unique<CalculatorContract>();
    cc->Initialize(calculator);
    auto abslStatus = mediapipe::OpenVINOInferenceCalculator::GetContract(cc.get());
    EXPECT_EQ(abslStatus.code(), expectedStatusCode) << abslStatus.message();
}
```